src/Uno.UWP/UI/Core/CorePhysicalKeyStatus.cs

```diff
@@ -1,5 +1,7 @@
 #nullable enable
 
+using System;
+
 namespace Windows.UI.Core
 {
 	public partial struct CorePhysicalKeyStatus
```

Comment:
@jeromelaban @spouliot Should these structs implement `IEquatable<T>`?

Reply:
AFAICT the original type does not implement the interface, so it's not required. It would be a problem for binary compatibility but I don't think it matter in this case.

https://learn.microsoft.com/en-us/uwp/api/windows.ui.core.corephysicalkeystatus?view=winrt-22621